deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+packageName: ibm-block-csi-operator
+channels:
+- name: stable
+  currentCSV: ibm-block-csi-operator.v1.1.0
+defaultChannel: stable

```

Comment:
Please don't create new folder 1.1.0 in this pr, as I told, I will create new ticket for it.

Reply:
Why do you think it should have been a different ticket?